include/ck/tensor_operation/gpu/element/element_wise_operation.hpp

```diff
@@ -1,97 +1,13 @@
 #pragma once
 #include "data_type.hpp"
 #include "math_v2.hpp"
+#include "unary_element_wise_operation.hpp"
+#include "binary_element_wise_operation.hpp"
 
 namespace ck {
 namespace tensor_operation {
 namespace element_wise {
 
-struct PassThrough
-{
-    __host__ __device__ void operator()(float& y, const float& x) const { y = x; }
```

Comment:
Use static_assert + is_same to replace overloadding

Reply:
Changed by  [1a4ee](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/274/commits/61d01e71055f76b34b5a5e3b9f064f6cfbf1a4ee)